processor/processor.go

```diff
@@ -327,6 +329,9 @@ func (p *processor) processType(pkg *loader.Package, parentType *types.Type, t g
 		if info != nil {
 			underlying = pkg.TypesInfo.TypeOf(info.RawSpec.Type)
 		}
+		if underlying.String() == "string" {
+			typeDef.Values = extractConstantsForAliasedString(pkg, typeDef.Name)
```

Comment:
Values sounds a bit generic. Would it be more explicit with `ConstantValues` to avoid confusion?
```suggestion
			typeDef.ConstantValues = lookupConstantValuesForAliasedType(pkg, typeDef.Name)
```